project-base/storefront/components/Layout/Header/Navigation/NavigationItem.tsx

```diff
@@ -12,22 +13,30 @@ type NavigationItemProps = {
 export const NavigationItem: FC<NavigationItemProps> = ({ navigationItem }) => {
     const [isMenuOpened, setIsMenuOpened] = useState(false);
     const hasChildren = !!navigationItem.categoriesByColumns.length;
+    const isMenuOpenedDelayed = useDebounce(isMenuOpened, 200);
 
     return (
         <li className="group" onMouseEnter={() => setIsMenuOpened(true)} onMouseLeave={() => setIsMenuOpened(false)}>
             <ExtendedNextLink
                 href={navigationItem.link}
                 className={twJoin(
-                    'relative m-0 flex items-center px-2 py-4 text-sm font-bold uppercase text-white no-underline hover:text-secondary hover:no-underline group-hover:text-secondary group-hover:no-underline vl:text-base',
+                    'relative m-0 flex items-center px-6 xl:px-5 py-4 group-first-of-type:pl-0 text-sm font-bold uppercase text-white hover:text-white no-underline hover:no-underline vl:text-base transition-colors',
+                    (isMenuOpenedDelayed || !hasChildren) &&
+                        'hover:text-secondary group-hover:text-secondary group-hover:no-underline',
```

Comment:
After changing these styles, you also have to change `NavigationPlaceholder`, because otherwise their styles do not match, which causes a layout shift:

https://github.com/user-attachments/assets/50eb7776-f9a3-4f82-a06b-2187002ad9d1